5-network/01-fetch/article.md

```diff
@@ -1,158 +1,158 @@
+let commits = await response.json(); // odczytaj ciało odpowiedzi i zwróć jako JSON
 */!*
 
 alert(commits[0].author.login);
 ```
 
-Or, the same without `await`, using pure promises syntax:
+Bądź to samo, ale bez `await`, a jedynie za pomocą czystej składni obietnic:
 
 ```js run
 fetch('https://api.github.com/repos/javascript-tutorial/en.javascript.info/commits')
   .then(response => response.json())
   .then(commits => alert(commits[0].author.login));
 ```
 
-To get the reponse text, `await response.text()` instead of `.json()`:
+Aby pobrać odpowiedź jako tekst, użyj `await response.text()` zamiast `.json()`:
 
 ```js run async
 let response = await fetch('https://api.github.com/repos/javascript-tutorial/en.javascript.info/commits');
 
-let text = await response.text(); // read response body as text
+let text = await response.text(); // odczytaj ciało odpowiedzi jako tekst
 
 alert(text.slice(0, 80) + '...');
 ```
 
-As a show-case for reading in binary format, let's fetch and show a logo image of ["fetch" specification](https://fetch.spec.whatwg.org) (see chapter [Blob](info:blob) for details about operations on `Blob`):
+Aby zaprezentować odczyt danych w formacie binarnym, pobierzmy obraz logo [specyfikacji "fetch"](https://fetch.spec.whatwg.org) (patrz rozdział [Blob](info:blob) odnośnie operacji na obiekcie typu `Blob`):
 
 ```js async run
 let response = await fetch('/article/fetch/logo-fetch.svg');
 
 *!*
-let blob = await response.blob(); // download as Blob object
+let blob = await response.blob(); // pobierz logo jako obiekt typu Blob
 */!*
 
-// create <img> for it
+// stwórz dla niego znacznik <img>
 let img = document.createElement('img');
 img.style = 'position:fixed;top:10px;left:10px;width:100px';
 document.body.append(img);
 
-// show it
+// wyświetl je
 img.src = URL.createObjectURL(blob);
 
-setTimeout(() => { // hide after three seconds
+setTimeout(() => { // ukryj po upływie trzech sekund
   img.remove();
   URL.revokeObjectURL(img.src);
 }, 3000);
 ```
 
 ````warn
-We can choose only one body-reading method.
+Można wybrać tyko jedną z metod odczytywania ciała odpowiedzi.
 
-If we've already got the response with `response.text()`, then `response.json()` won't work, as the body content has already been processed.
+Jeśli już zdecydowaliśmy się na `response.text()`, wówczas `response.json()` nie zadziała, ponieważ zawartość ciała odpowiedzi została już wcześniej przetworzona.
 
 ```js
-let text = await response.text(); // response body consumed
-let parsed = await response.json(); // fails (already consumed)
+let text = await response.text(); // ciało odpowiedzi zostaje przetworzone
+let parsed = await response.json(); // nie powiedzie się (przetworzone wcześniej)
 ````
 
-## Response headers
+## Nagłówki odpowiedzi
 
-The response headers are available in a Map-like headers object in `response.headers`.
+Nagłówki odpowiedzi są dostępne w obiekcie nagłówków podobnym do obiektu Map, a konkretnie w `response.headers`.
```

Comment:
```suggestion
Nagłówki odpowiedzi są dostępne w obiekcie nagłówków podobnym do obiektu klasy Map, pod właściwością `response.headers`.
```

Reply:
W popularnej dokumentacji, typu MDN nie znalazłem w sumie słowa `class` nigdzie przy `Map`. Nie wiem czy powinniśmy to tutaj wprowadzać, zwłaszcza, że instancje `Map` dziedziczą po `Map.prototype`